### PR TITLE
DCS-894 updated header

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/services/userService.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ module.exports = (nomisClientBuilder) => {
       isPrisonUser,
       activeCaseLoad,
       activeCaseLoadId,
+      displayNameInitial: `${forenameToInitial(profile.name)}`,
```
Comment (louisenorris) [inline server/services/userService.js:29]: Just to check - do we know that a user will always have a name in profile for us to use in forenameToInitial?

Comment (andrewrlee) [reply server/services/userService.js:29]: should do

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/app.js [hunk 2]
```diff
@@ -208,12 +208,22 @@ module.exports = function createApp({
...
       hostname: req.hostname,
+      authUrl: config.nomis.authUrl,
+      apiClientId: config.nomis.apiClientId,
     }
     next()
   }
 
   app.use(addTemplateVariables)
 
+  function addreturnURL(req, res, next) {
```
Comment (andrewrlee) [inline server/app.js:219]: super meh.. but `R` for `return`

Comment (louisenorris) [reply server/app.js:219]: pushing up a correction for this now - thought I might as well.

**Comment by andrewrlee** [comment]
closing as superseded

**Pull request closed without merging**